src/app/jscaip/BoardValue.ts

```diff
@@ -3,6 +3,8 @@ import { Player, PlayerOrNone } from './Player';
 export class BoardValue {
 
     public static fromWinner(player: PlayerOrNone): BoardValue {
+        // TODO FOR REVIEW: j'ai un peu autisté là pour le coup avec from=fallible mais ofWinner c'est bizarre
+        // Et deux trois autres modif que j'ai faite ont pu être pareille, sorry :D
```

Comment:
faut préfixer toutes les lignes de commentaire like this avec du TODO pour éviter qu'une survive !